cssselect/xpath.py

```diff
@@ -308,10 +308,12 @@ def xpath_attrib(self, selector):
             attrib = '@' + name
         else:
             attrib = 'attribute::*[name() = %s]' % self.xpath_literal(name)
-        if self.lower_case_attribute_values:
-            value = selector.value.lower()
+        if selector.value is None:
+            value = None
+        elif self.lower_case_attribute_values:
+            value = selector.value.value.lower()
```

Comment:
it's the symmetrical change for this: https://github.com/scrapy/cssselect/pull/75/files/08b0e472175197d4887612cd90d74815ab10d90a#diff-adc3ae8f2cf8b1931771d84ca7af6275R596

needed because i had to distinguish between different types of values here: https://github.com/scrapy/cssselect/pull/75/files/08b0e472175197d4887612cd90d74815ab10d90a#diff-adc3ae8f2cf8b1931771d84ca7af6275R651 (strings have to be serialised with quotes, number have to skip them, …)

I'm not sure I remember why I needed